### ISSUE TITLE
New Feature Request: More Options for Multiplayer Game Modes

### ISSUE THREAD

**Issue by kent1313** [supertuxkart/stk-code#4979]
## Description
I love to play Super Tux Kart with my friends, but with only a couple players, it would be nice to battle bots in an online game. For multiplayer game play, there is an option to add up to 7 bots in a normal race, but you cannot add bots to a battle or soccer game. The following extra features for multiplayer games would make games with just a few people a lot of fun:
- Option to add bots for multiplayer battles and soccer games
- Multiplayer follow the leader game mode
- Multiplayer three strikes battle game mode

## Configuration
STK release version: 1.3

**Comment by Lulu-Games** [comment]
Good idea, I know that issue I have got the same problem. Your release is 1.3 but there is a 1.4 out now

**Comment by kent1313** [comment]
Cool! I installed Super Tux Kart from the Ubuntu App Manager, so that make sense that I don't have the latest version. Thanks for letting me know about the newer version! I am now on version 1.4.

It's good to know that other users would enjoy more multiplayer options as well. I would love to see this added to Super Tux Kart!

**Comment by Lulu-Games** [comment]
You are using Linux? Cool! But why don't you use simply: `sudo apt install supertuxkart` in a terminal? Wich user interface do you use? (e.g. KDE Plasma; GNOME; LXDE...)
